docs/internal/contributing/contributing-to-helm-chart.md

```diff
@@ -62,8 +62,11 @@ In order to keep the kustomize configuration manageable, it is divided into over
 
 For example, at the time of writing, the Helm chart is passed through the following overlays:
 
+```bash
```

Comment:
I had to fix this because now the Helm Chart linter starts linting this markdown file.